launch-or-update-azure.sh

```diff
@@ -332,12 +332,14 @@ helm upgrade bc $chart -n $namespace -i -f environments/azure.yaml \
   --set of.ingress.hosts[0].host=$basePath \
   --set redis-commander.ingress.hosts[0].host=$basePath \
   --set azurite.connectionString="$azureStorageConnectionString" \
-  --set file-loader.azureBlobStorage.containerName=$containerName \
-  --set dapi.azureBlobStorage.containerName=$containerName \
   --set redis.disabledUrl=$redisServer \
   --set redisPassword="$redisPassword" \
+  --set containerName="$containerName" \
   $versionArg \
   $waitArg \
   $emailArg \
   $helmUpgradeArgs \
-  $hostAliases
\ No newline at end of file
+  $hostAliases
+
+#  --set file-loader.azureBlobStorage.containerName=$containerName \
+#  --set dapi.azureBlobStorage.containerName=$containerName \
```

Comment:
I think we can get rid of these lines now.

Reply:
Removed as requested